apisix/http/route.lua

```diff
@@ -31,6 +31,23 @@ local loadstring = loadstring
 local _M = {}
 
 
+function _M.prefix_uris(route, service)
+    if service and service.value.path_prefix then
+        local uri, uris
+        if route.value.uris then
+            uris = core.table.new(#route.value.uris)
+            for _, uri in ipairs(route.value.uris) do
+                core.table.insert(uris, service.value.path_prefix .. uri)
```

Comment:
A tiny detail: what if the path_prefix is ended with `/`, in such a case, we'll have continuous slashes. E.g., `/api/v1//order`. While this affects the route match?

Reply:
That depends on the validation requirements.
Not only the trailing slash, the slash, and other invalid chars may exist in any position of the string. So if we need real validation, we need to define its requirement.
That is, is it enough if I just remove the trailing slash if exists?